src/libraries/System.Net.Quic/src/Resources/Strings.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 
+    Microsoft ResX Schema 
```

Comment:
this seems wrong e.g. bunch of spaces added to where they should not be.

Reply:
Sorry about that, I think this was done by my IDE. I've removed all the random Whitespace changes.